fairlearn/preprocessing/_correlation_remover.py

```diff
@@ -56,6 +56,11 @@ class CorrelationRemover(BaseEstimator, TransformerMixin):
     Note that the lack of correlation does not imply anything about statistical dependence.
     Therefore, we expect this to be most appropriate as a preprocessing step for
     (generalized) linear models.
+
+    .. versionadded:: 0.6
+
+    .. versionchanged:: 0.8
```

Comment:
IMO this isn't a change that is user-facing so we should omit it and just keep versionadded. @adrinjalali  made that change for sklearn compatibility and should probably decide 🙂

Reply:
Yes, things which are not _public_ don't have to have a version added or changed tag.